### ISSUE TITLE
bug(gui-client): Clearing logs doesn't clear IPC service logs on Windows

### ISSUE THREAD

**Issue by ReactorScram** [CloudFire-LLC/cloudfire#5257]
Add this to the smoke test after the zip export.

I think it's cause of the permissions. On Linux I have a special thing for it: https://github.com/firezone/firezone/blob/c35a7579a802edb9658201dfcc070fe46d514e2e/rust/gui-client/src-tauri/deb_files/firezone-client-ipc.service#L10-L11

If #5279 merged, we could send a signal over IPC, but that one's blocked on a failing test

**Comment by ReactorScram** [comment]
Closing in favor of #5453
